modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/freelist/PagesListView.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.persistence.freelist;
+
+import org.apache.ignite.internal.managers.systemview.walker.ViewAttribute;
+import org.apache.ignite.internal.pagemem.PageIdUtils;
+import org.apache.ignite.spi.systemview.view.SystemView;
+
+/**
+ * Pages-list representation for a {@link SystemView}.
+ */
+public class PagesListView {
```

Comment:
Let's place this class with the other system view row class in package `org.apache.ignite.spi.systemview.view`

Reply:
Most of the fields accessed from PagesListView are protected or package private. I think it's better to put class in the same package than increase access level for these fields.

Reply:
You can provide getters for these values.